### PR TITLE
Add defer and lab to errors

### PR DESCRIPTION
**Pull request by juliaogris**

### CODE REVIEW

File: content/gotraining.slide [hunk 1]
```diff
@@ -577,12 +577,23 @@ Use `panic` rarely and judiciously in your own code!
 
 - [[https://goplay.space/#h1kA3qKaO6B][Errors]]
 - [[https://goplay.space/#9vu6iJHXMHC][Panic]]
-- [[][Defer and Recover]]
+- [[https://goplay.space/#pMbrba6y9cY][Defer]]
```
Comment (camscale) [inline content/gotraining.slide]: Wow. that defer sample is perfect. Well done!

In all seriousness, I'd also use your other one too. This sample visually shows how defer works. Your copyFile one is more realistic and attaches the defer resource release to the resource acquisition, which is the most common pattern.

https://goplay.space/#pc68j1BPMxn

File: content/gotraining.slide [hunk 2]
```diff
@@ -577,12 +577,23 @@ Use `panic` rarely and judiciously in your own code!
...
 
-* Lab 7
+* Lab 7 - Errors
 
-TODO
+- Create an executable go program in directory `7_errors`
+- Copy the CRUD puppy from upstream master
+- Add a custom error type `Error` with fields `Message` and `Code`
+- Extend the `Storer` interface for all methods to also return `error`
+- Create errors for
```
Comment (camscale) [inline content/gotraining.slide]: nit: colon at end of line

Comment (juliaogris) [reply content/gotraining.slide]: Done.

File: content/gotraining.slide (hunk 1 above)
Comment (juliaogris) [reply content/gotraining.slide]: Done.

**Comment by juliaogris** [comment]
@camh-anz - whenever you have a moment, PITAL!

**Review by camscale — APPROVED** [review]
LGTM

**Pull request merged**